src/pages/workspaces/workspace/Container.js

```diff
@@ -113,7 +114,7 @@ export class WorkspaceContainer extends Component {
       contextBar({
         style: {
           paddingLeft: '5rem', borderBottom: `5px solid ${Style.colors.secondary}`,
-          textAlign: 'center', color: 'white', lineHeight: '3.5rem', textTransform: 'uppercase'
+          textAlign: 'center', color: 'white', lineHeight: '3.75rem', textTransform: 'uppercase'
```

Comment:
The only issue here is that the options in the side nav no longer align with the bar height, so you should tweak those too.

Reply:
I didn't realize that was deliberate. I'll patch it.